Source/WebCore/Modules/mediastream/gstreamer/GStreamerStatsCollector.cpp

```diff
@@ -186,7 +186,7 @@ RTCStatsReport::PeerConnectionStats::PeerConnectionStats(const GstStructure* str
 
 RTCStatsReport::TransportStats::TransportStats(const GstStructure* structure)
     : Stats(Type::Transport, structure)
-    , selectedCandidatePairId(gstStructureGetString(structure, "selected-candidate-pair-id"_s).toString())
+    , selectedCandidatePairId(String(gstStructureGetString(structure, "selected-candidate-pair-id"_s).span()))
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?